data/test-cases/plugins/zoom/general/Ephemeral_post_for_PM_disabled.md

```diff
@@ -0,0 +1,83 @@
+---
+# (Required) Ensure all values are filled up
+name: "New meeting is being created after clicking `Create new meeting` in the threads view. "
+status: Active
+priority: Normal
+folder: General
+authors: "@arush-vashishtha"
+team_ownership: []
+priority_p1_to_p4: P2 - Core Functions (Do core functions work?)
+
+# (Optional)
+location: null
+component: null
+tags: []
+labels: []
+tested_by_contributor: null
+
+# (Optional) Test type and tools
+cypress: null
+detox: null
+mmctl: null
+playwright: null
+rainforest: []
+manual_test_environments: []
+
+# Do not change
+id: null
+key: null
+created_on: null
+last_updated: null
+case_hashed: null
+steps_hashed: null
+---
+
+**Step 1**
+
+1. Go to your Zoom settings and disable `Personal Meeting ID` on Zoom.
```

Comment:
Do we need to go to Zoom to perform this, or the Zoom plugin configs?

Reply:
We need to go to the [Zoom](https://zoom.us/profile/setting?_x_zm_rtaid=6m8hP3gKQSGXTALigy0X5Q.1705649510753.a673ff29bd8b7130b6232ec000939871&_x_zm_rhtaid=615) and disable the PMI there